index.html

```diff
@@ -13,7 +13,7 @@
                 <a href="/Users/maksim/Desktop/project/about.html">Contact</a>

```

Comment:
better way to show this is nav => ul => li > a. because we in any way have list of liks here